server/train/train_types.py

```diff
@@ -10,14 +10,16 @@
 
 import enum
 import random
+from typing import List
 
 SYSTEM_FEATURES = ["cpu_architecture"]
 
 COUNTER_FEAUTRES = ["cache_miss", "cpu_cycles", "cpu_instr"]
 CGROUP_FEATURES = ["cgroupfs_cpu_usage_us", "cgroupfs_memory_usage_bytes", "cgroupfs_system_cpu_usage_us", "cgroupfs_user_cpu_usage_us"]
 IO_FEATURES = ["bytes_read", "bytes_writes"]
 BPF_FEATURES = ["cpu_time"]
-KUBELET_FEATURES =['kubelet_memory_bytes', 'kubelet_cpu_usage']
+KUBELET_FEATURES = ['kubelet_memory_bytes', 'kubelet_cpu_usage']
+IRQ_FEATURES = ["block_irq", "net_tx_irq", "net_rx_irq"]
```

Comment:
since `cpu_time` and all irq metrics are all from ebpf program, we can consolidate them into the new `BPF_FEATURES`

Reply:
Ok I will remove IRQ Features and instead relocate these metrics into BPF Features